.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "friday"
+
+  # Updates the dependencies of the GitHub Actions workflows
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
why not weekly like `pip`?

Reply:
thought it to be less important that way it may reduce the traffic a bit. but I'm open to setting it to weekly as well

Reply:
okay so dependabot will update `requirements.txt` for `pip` and will update which file for `github-actions`? Can we also make it update `pre-commit-config`?

Reply:
Yes, it will even track more files - [this table] lists `pipfile(.lock)` and `setup.py` and for [my repo](https://github.com/Bibo-Joshi/led-strip-controller/network/updates), the insights/dep graph/dependabot also lists `pyproject.toml`, `reqs-dev.txt` and `setup.cfg`.

for `pre-commit` I don't find anything in the [supported ecosystems](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#supported-package-ecosystems). We could consider switching to the pre-commit.ci for the `pre-commit` tests - maybe another item for #2925 :D